src/videoserver/apps/projects/routes.py

```diff
@@ -1230,9 +1230,12 @@ def get(self, project_id):
         video_range = request.headers.environ.get('HTTP_RANGE')
         length = self.project['metadata'].get('size')
         if video_range:
-            start = int(re.split('[= | -]', video_range)[1])
+            _range = re.split('[= | -]', video_range)
+            start = int(_range[1])
             # TODO doublecheck streaming range
             end = length - 1
+            if len(_range) == 3 and _range[2]:
```

Comment:
Guys, please, add a comment for this part. I'm sure in the future it would be easier to understand what is happening here :)